src/components/FetchHandler.tsx

```diff
@@ -27,28 +25,27 @@ export const FetchHandler = <TResult extends {}>({
     notFoundIndicator,
     loadingIndicator,
 }: Props<TResult>) => {
-    const { loading, data, error, trigger, errorCode } = usePromise(apiMethod);
-    const [dependencyList, setDependencyList] = useState(undefined);
+    const { loading, data, error, trigger } = usePromise(apiMethod);
 
     useEffect(() => {
         trigger();
-    }, [dependencyList, trigger]);
+    }, [trigger]);
 
     const renderError = () => {
-        if (errorIndicator && (errorCode !== 404 && notFoundIndicator)) {
+        if (errorIndicator && error?.code !== 404 && notFoundIndicator) {
             return errorIndicator;
-        } else if (errorCode === 404 && notFoundIndicator) {
+        } else if (error?.code === 404 && notFoundIndicator) {
             return notFoundIndicator;
         }
 
-        return errorCode !== 401 ? <Alert type="error" message={errorText} /> : <Spinner centered />;
+        return error?.code !== 401 ? <Alert type="error" message={errorText} /> : <Spinner centered />;
     };
 
     return (
         <>
             {loading && (loadingIndicator ? loadingIndicator : <Spinner centered />)}
             {error && renderError()}
-            {data && children(data, (dependencies: any) => setDependencyList(dependencies))}
```

Comment:
varför vill du inte ha kvar den biten?

Reply:
"We can tweak our API to leverage both default children and children as an object map, so consumer would be able to use one or another."

https://medium.com/@martin_hotell/react-children-composition-patterns-with-typescript-56dfc8923c64